### PR TITLE
Added icons to main settings screen

### PR DESCRIPTION
**Pull request by ByteHamster**
Updated the main settings screen with icons

### CODE REVIEW

File: app/ui/src/main/java/com/fsck/k9/ui/settings/SettingsActionItem.kt [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
-import kotlinx.android.synthetic.main.text_list_item.*
+import kotlinx.android.synthetic.main.text_icon_list_item.*
+import android.util.TypedValue
+
+
 
 internal class SettingsActionItem(val text: String, val action: SettingsAction) : Item() {
 
-    override fun getLayout(): Int = R.layout.text_list_item
+    @AttrRes var icon = R.attr.iconActionSettings
```
Comment (cketti) [inline app/ui/src/main/java/com/fsck/k9/ui/settings/SettingsActionItem.kt]: Please make this a read-only property (`val`) declared in the constructor.

File: app/ui/src/main/res/layout/account_list_item.xml [hunk 2]
```diff
@@ -1,37 +1,48 @@
...
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:background="?android:attr/selectableItemBackground"
-    android:clickable="true"
-    android:focusable="true"
-    android:gravity="center_vertical"
-    android:minHeight="?android:attr/listPreferredItemHeightSmall"
-    android:orientation="vertical">
+              xmlns:tools="http://schemas.android.com/tools"
```
Comment (cketti) [inline app/ui/src/main/res/layout/account_list_item.xml]: We use 4 spaces for indentation in XML files. Please adjust your IDE's code formatting settings accordingly.

**Comment by cketti** [comment]
Settings screen of the Outlook app:
screenshot_outlook_settings

File: app/ui/src/main/res/layout/account_list_item.xml (hunk 2 above)
Comment (ByteHamster) [reply app/ui/src/main/res/layout/account_list_item.xml]: I already use tabs. Android Studio's auto formatter just uses more spaces than originally used in this file

**Comment by ByteHamster** [comment]
@cketti sorry for the mess. It would probably be a good idea to squash this while merging to master.

**Pull request merged**